src/FishDeskNextReborn/window/FlitterWindow.xaml

```diff
@@ -11,15 +11,17 @@
     <Grid>
        
         <StackPanel Orientation="Vertical">
-            <TextBlock Margin="10" FontSize="20">过滤管理器</TextBlock>
-            <TextBlock Margin="5">某些应用程序存在单实例启动，在其他桌面启动这些程序时会导致Windows Shell自动切换桌面。过滤器会尝试杀死以下注册的程序。(一行一个)</TextBlock>
+            <TextBlock Margin="5" FontSize="20">过滤管理器</TextBlock>
+            <TextBlock Margin="10">某些应用程序存在单实例启动的行为，在其他桌面启动时，Windows Shell将会自动切换桌面。</TextBlock>
+            <TextBlock Margin="10">过滤器会尝试杀死以下注册的程序。</TextBlock>
+            <TextBlock Margin="10">列表中的每一行填写一个程序。</TextBlock>
             <TextBox x:Name="ProgListTxtbox" TextWrapping="Wrap" Height="160" Margin="10"></TextBox>
             <StackPanel Orientation="Horizontal">
-                <Button x:Name="LoadBtn" Margin="5">加载/刷新</Button>
+                <Button x:Name="LoadBtn" Margin="15">刷新</Button>
```

Comment:
https://learn.microsoft.com/zh-cn/dotnet/desktop/wpf/advanced/alignment-margins-and-padding-overview?view=netframeworkdesktop-4.8#understanding-margin-properties
Try Margin "15 5 0 5"